Moose Development/Moose/Core/Zone.lua

```diff
@@ -1909,47 +2013,86 @@ function ZONE_POLYGON_BASE:GetBoundingSquare()
   local y1 = self._.Polygon[1].y
   local x2 = self._.Polygon[1].x
   local y2 = self._.Polygon[1].y
-  
+
   for i = 2, #self._.Polygon do
     self:T2( { self._.Polygon[i], x1, y1, x2, y2 } )
     x1 = ( x1 > self._.Polygon[i].x ) and self._.Polygon[i].x or x1
     x2 = ( x2 < self._.Polygon[i].x ) and self._.Polygon[i].x or x2
     y1 = ( y1 > self._.Polygon[i].y ) and self._.Polygon[i].y or y1
     y2 = ( y2 < self._.Polygon[i].y ) and self._.Polygon[i].y or y2
-    
+
   end
 
   return { x1 = x1, y1 = y1, x2 = x2, y2 = y2 }
 end
 
+--- Draw a frontier on the F10 map with small filled circles.
+-- @param #ZONE_POLYGON_BASE self
+-- @param #number Coalition (Optional) Coalition: All=-1, Neutral=0, Red=1, Blue=2. Default -1= All.
+-- @param #table Color (Optional) RGB color table {r, g, b}, e.g. {1, 0, 0} for red. Default {1, 1, 1}= White.
+-- @param #number Radius (Optional) Radius of the circles in meters. Default 1000.
+-- @param #number Alpha (Optional) Alpha transparency [0,1]. Default 1.
+-- @param #number Segments (Optional) Number of segments within boundary line. Default 10.
+-- @param #boolean Closed (Optional) Link the last point with the first one to obtain a closed boundary. Default false
+-- @return #ZONE_POLYGON_BASE self
+function ZONE_POLYGON_BASE:Boundary(Coalition, Color, Radius, Alpha, Segments, Closed)
+    Coalition = Coalition or -1
+    Color = Color or {1, 1, 1}
+    Radius = Radius or 100
```

Comment:
Default Radius value should be 1000 according to the luaDoc

Reply:
I’m going to guess the documentation is wrong because 100m little circles are probably intended as default. 1km circles for this sound way too big.

Reply:
I written the function, 1000m was a pretty good default value considering the default F10 map zoom.
But I guess it's your call now. (1km is not that big).

Reply:
Ah, then whatever works well. I haven't had a chance to actually look at it but if 1000m is good then just go with it. Whatever the initial intention was either fix the docs or the code and I'm good with it.